pandas/core/apply.py

```diff
@@ -1832,12 +1832,12 @@ def warn_alias_replacement(
         full_alias = alias
     else:
         full_alias = f"{type(obj).__name__}.{alias}"
-        alias = f"'{alias}'"
+        alias = f'`"{alias}"`'
```

Comment:
Might be good to remove the ``` ` ```s as they might not always render like markdown for example

Reply:
Updated @mroeschke